src/config/dynamic-theme-fixes.config

```diff
@@ -1638,6 +1638,23 @@ path[class^="wavePath"] {
 
 ================================
 
+discover.manjaro.org
+
+CSS
```

Comment:
This fix can be removed the `IGNORE IMAGE ANALYSIS` will do it's work and make it visible.

Reply:
Does not work [here](https://discover.manjaro.org/applications/abiword) for instance if I remove the CSS fix